histomicstk/AreaOpenLabel.py

```diff
@@ -0,0 +1,58 @@
+import histomicstk as htk
+import numpy as np
+import scipy.ndimage.measurements as ms
+
+
+def AreaOpenLabel(Label, Area):
+    """Removes small objects from label image.
+
+    Parameters:
+    -----------
+    Label : array_like
+        A uint32 type label image generated by segmentation methods.
+    Area : int
+        Area threshold for objects. Objects with fewer than 'Area' pixels will
+        be zeroed to merge with background.
+
+    Notes:
+    ------
+    Objects are assumed to have positive nonzero values. Label image will be
+    condensed during processing.
+
+    Returns:
+    --------
+    Split : array_like
+        A uint32 label where objects with pixels < Area are removed.
+
+    See Also:
+    ---------
+    CondenseLabel, ShuffleLabel, SplitLabel, MaxwidthOpenLabel
+    """
+
+    # copy input image
+    Opened = Label.copy()
+
+    # condense label image
+    if np.unique(Opened).size-1 != Opened.max():
```

Comment:
Yes we should. Condensing the label image is also not really needed - it's more of a convenience for implementation. I plan to re-write so that we don't need to call condense because it completely breaks the correspondence of values between inputs and outputs.